test/unit/collector/api-connect.test.js

```diff
@@ -243,91 +243,6 @@ tap.test('succeeds when given a different port number for redirect', (t) => {
   })
 })
 
-tap.test('succeeds after one 503 on preconnect', (t) => {
```

Comment:
did I do this or was it you @jmartin4563 ? this still seems legit

Reply:
I removed this one, because the test right below it tests the exact same thing, just after five attempts. I can readd if you think it's still useful to have the one vs several attempts?

Reply:
my only concern is the other test has this ```// TODO: 503 tests can likely be consolidated into single test func
// passed to t.test() while specifying different # of 503s.
```

we should probably just refactor to address this todo